<?xml version="1.0"?>

<launch>

  <node name="mir_pose_unidriver" pkg="unification_roscontrol" type="mir_pose_unidriver.py">
  </node>
  <node name="mir_mode_unidriver" pkg="unification_roscontrol" type="mir_mode_unidriver.py">
  </node>
  <node name="ur_pose_unidriver" pkg="unification_roscontrol" type="ur_pose_unidriver.py">
  </node>
  <node name="ur_mode_unidriver" pkg="unification_roscontrol" type="ur_mode_unidriver.py">
  </node>
  <node name="agv_alvar_unidriver" pkg="unification_roscontrol" type="agv_alvar_unidriver.py">
  </node>

  <include file="$(find unification_roscontrol)/launch/remote_recu_unidriver.launch" />

  <include file="$(find unification_roscontrol)/launch/remote_aecu_unidriver.launch" />

  <include file="$(find unification_roscontrol)/launch/remote_hecu_unidriver.launch" />
  
</launch>